extensions/transactions/simple/standard/requirements/features/crs/REQ_other-crs.adoc

```diff
@@ -0,0 +1,7 @@
+[[req_simpletx_crs_other-crs]]
+[width="90%",cols="2,6a"]
+|===
+^|*Requirement {counter:req-id}* |*/req/simpletx/crs/other-crs*
+^|A |If a server implementing this standard also supports the http://fix.me[OGC API - Features - Part 2: Coordinate Reference Systems by Reference] extension, then all geometry-valued feature proprties presented to the server in a request body shall be expressed using a mutually agreeable crs that has been negotiated as per https://github.com/opengeospatial/conneg-by-crs/blob/master/18-072-Content-Negotiation-by-CRS.md[Negotiating Content by Coordinate Reference System (CRS)] for the collection being operated on.
```

Comment:
We cannot normatively reference this document, which has no status and an unclear future (last commit 1+ year old).